terraform/datadog/certificates.tf

```diff
@@ -0,0 +1,18 @@
+resource "datadog_monitor" "invalid_tls_cert" {
+  name              = "${format("%s Invalid TLS/SSL Certificate", var.env)}"
+  type              = "metric alert"
+  message           = "${format("{{hostname.name}} certificate {{#is_alert}}is invalid!{{/is_alert}}{{#is_warning}}expires in {{value}} days{{/is_warning}}\n\nSee [Team Manual > Responding to alerts > Invalid Certificates](%s#invalid-certificates) for more info.", var.datadog_documentation_url)}"
+  notify_no_data    = true
+  no_data_timeframe = 120
+
+  query = "${format("min(last_1h):min:tls.certificates.validity{deploy_env:%s} by {hostname} <= 0", var.env)}"
+
+  require_full_window = true
+
+  thresholds {
+    warning  = 7
```

Comment:
Is seven days enough? Any thoughts @alext ?

Reply:
I agree that this is a little short. I reckon we should trigger warning at 30 days, and critical at 7 days. We can always tune these later (different values may be appropriate if we switch to something with auto-renewing like ACM, but these seems sensible while it's a manual process.).